src/components/AttachmentPicker/index.js

```diff
@@ -35,11 +35,18 @@ class AttachmentPicker extends React.Component {
                 <input
                     hidden
                     type="file"
+                    accept="audio/*;video/*;image/*;application/pdf"
```

Comment:
If we fine tune it - it can filter/suggest files based on this attribute: 
![image](https://user-images.githubusercontent.com/12156624/113751860-093c8280-9715-11eb-8ba0-cd36d981074f.png)

I don't think it does anything at the moment, with or without it there are no restrictions but I'll just remove it to be safe